airbyte-webapp/src/components/ReleaseStageBadge/ReleaseStageBadge.tsx

```diff
@@ -37,13 +33,10 @@ export const ReleaseStageBadge: React.FC<ReleaseStageBadgeProps> = ({
       <div
         className={classNames(styles.pill, {
           [styles["pill--small"]]: small,
-          [styles["pill--contains-tag"]]: showFreeTag,
```

Comment:
One downside of putting all code shared with OSS on the other side of a component boundary: we no longer have a good way to conditionally change the `padding-right` from `6px` to `2px` based on whether a free tag is displayed. Instead, the `FreeTag` component uses a negative `margin-right`; that's a hacky solution, though, do let me know if you have a better idea.

Reply:
I do like that a bit better, but I'm inclined to save that for later, when consolidating the three implementations of stage pills (there's _another_ copy-paste `StageLabel` component in `cloud/components/experiments/` , used by the sign up page; I left that untouched for now since not-yet-created accounts will never be enrolled). If the overall design of the status pills changes, as things stand, it would be way too easy to overlook one of those.